### ISSUE TITLE
first questions :)

### ISSUE THREAD

**Issue by raveit65** [zhuyaliang/group-service#1]
I have installed it to `/usr`, but how can i test it?
```
[rave@mother ~]$ /usr/libexec/group-admin-daemon 
Lost Lost !!!!
```

**Comment by raveit65** [comment]
I tried it as normal user.
```
[rave@mother ~]$ /usr/libexec/group-admin-daemon
Lost Lost !!!!
[rave@mother ~]$ ps -efH | grep group-admin
rave      3688  3594  0 10:31 pts/0    00:00:00               grep --color=auto group-admin
[rave@mother ~]$ su
Passwort: 
[root@mother rave]# ps -efH | grep group-admin
root      3729  3698  0 10:31 pts/0    00:00:00                   grep --color=auto group-admin
```
As you can see there is no process.
But in `d-feet` i see `org.group.admin`
And after running `d-feet` the process is there.
```
[rave@mother ~]$ ps -efH | grep group-admin
rave      4276  4241  0 10:34 pts/1    00:00:00               grep --color=auto group-admin
root      4226     1  0 10:33 ?        00:00:00   /usr/libexec/group-admin-daemon
```
Is the expected behavior?

**Comment by zhuyaliang** [comment]
@raveit65 
You should use the root user to start it.
``  sudo /usr/libexec/group-admin-daemon ``
Running ``d-feet`` automatically starts the service,That's right.

**Comment by raveit65** [comment]
Ok, starting as root works.

**Comment by raveit65** [comment]
@zhuyaliang 
It's the first time that i package a systemd unit file for a RPM package.
So it is a D-Bus service per definition?
See https://fedoraproject.org/wiki/Packaging:Systemd#Definitions
And which kind of `activation` it is using, `DBus activation`?
See https://fedoraproject.org/wiki/Packaging:Systemd#Activation

I guess the service should be enabled as default?
see https://docs.fedoraproject.org/en-US/packaging-guidelines/DefaultServices/
For this i have to ask ferora-release-engineering team

**Comment by zhuyaliang** [comment]
@raveit65 
You're right. It's a D-Bus service  per definition，It uses D-Bus activation，You don't have to change any options and configuration files, just install the files in the specified directory according to data/meson.build
